lib/web/mage/validation.js

```diff
@@ -396,6 +396,24 @@
             $.mage.__('Please enter at least {0} characters')
         ],
 
+        /* detect chars that would require more than 3 bytes */
+        'charbytes': [
```

Comment:
What `charbytes`  stands for? I would call it something like `validate-no-utf8mb4-characters` for consistency with other rules.

Reply:
Good?